### PR TITLE
[12.0][MIG] Migrate base_suspend_security 

### PR DESCRIPTION
**Pull request by RobinetDenisAcsone**
Summary of changes:
* README fragments
* version number

### CODE REVIEW

**Comment by RobinetDenisAcsone** [comment]
Travis is red because of the readme of base_user_role, fixed in https://github.com/OCA/server-backend/pull/23

**Review by ThomasBinsfeld — APPROVED** [review]

**Comment by ThomasBinsfeld** [comment]
@RobinetDenisAcsone This one can be rebased.

File: base_suspend_security/__init__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (hbrunn) [inline base_suspend_security/__init__.py]: please change this to short headers, also in the following files

**Review by hbrunn — APPROVED** [review]
thanks!

**Comment by pedrobaeza** [comment]
You should take into account https://github.com/OCA/server-backend/commit/4ee3ee65c41e994301a9831e8433b8e66eb6e49f that I was forced to put directly in 11.0 for not breaking totally the service.

File: base_suspend_security/__init__.py (hunk 1 above)
Comment (RobinetDenisAcsone) [reply base_suspend_security/__init__.py]: done

**Comment by RobinetDenisAcsone** [comment]
@pedrobaeza the _browse change isn't done (yet?) in [12.0](https://github.com/odoo/odoo/blob/12.0/odoo/models.py#L4615) and [master](https://github.com/odoo/odoo/blob/master/odoo/models.py#L4681)

**Comment by pedrobaeza** [comment]
Uhm, I see. It's a matter of time then...

**Comment by hbrunn** [comment]
@RobinetDenisAcsone now the patch has landed. Could you apply @pedrobaeza's commit and then we can merge?

**Comment by RobinetDenisAcsone** [comment]
@pedrobaeza @hbrunn commit cherry picked, should be ready to be merged.

**Review by ThomasBinsfeld — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@hbrunn please remember to check the line in the migration issue when merging migration PRs. I have done it already for this one.

**Comment by hbrunn** [comment]
ah, thanks. shouldn't we automate that away?

**Comment by pedrobaeza** [comment]
Yes, see https://github.com/OCA/oca-github-bot/issues/32 for one possible specification.

**Pull request merged**